### PR TITLE
{chem}[GCCcore/11.3.0] Simple-DFTD3 v0.7.0, mstore v0.2.0, mctc-lib v0.3.1, ...

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by seb45tian** [comment]
Test report by @seb45tian
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
r10n47.palma.wwu - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d8a9bdb648ed12b49ec2aa7c8b79e36f for a full test report.

**Comment by sassy-crick** [comment]
Suggestion for Simple-DFTD3: Using `Ninja` to build it maybe? Have a look what I have done in PR #16889 and I would suggest to add the references as well for example.

**Comment by lexming** [comment]
@sassy-crick thanks for the feedback! updated with your suggestions

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

File: easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb:1]: @lexming please reinstate the author attribution

Comment (verdurin) [reply easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb:1]: I know you've made some changes, but still...

File: easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    ('mstore', '0.2.0'),
+    ('TOML-Fortran', '0.3.1'),
+]
+
+build_shared_libs = True
+
+configopts = '-DWITH_BLAS=1 -DWITH_OpenMP=1'
+
+test_cmd = 'ctest'
+runtest = ''
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022a.eb:38]: Could you add a comment explaining the need for this?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/m/mstore/mstore-0.2.0-GCCcore-11.3.0.eb:3]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16760/commits/8357b48c2f784b23a7b4d414c648d78ec83f5edf

File: easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022a.eb (hunk 2 above)
Comment (lexming) [reply easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022a.eb:39]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16760/commits/a44e30ce8513cf5ff02978fb426c5f970c16ac73

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/cca6ade0135725b32a13b12862ab3479 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**